leaflet-markercluster/index.d.ts

```diff
@@ -6,138 +6,161 @@
+        * Use if you are using big marker icons (Default: 1).
+        */
+        spiderfyDistanceMultiplier?: number;
+
+        /*
+        * Function used to create the cluster icon
+        */
+        iconCreateFunction?: ((cluster: L.MarkerCluster) => L.Icon);
+
+        /*
+        * Boolean to split the addLayers processing in to small intervals so that the page does not freeze.
+        */
+        chunkedLoading?: boolean;
+
+        /*
+        * Time delay (in ms) between consecutive periods of processing for addLayers. Default to 50ms.
+        */
+        chunkDelay?: number;
     }
 
     export interface MarkerClusterGroup extends FeatureGroup {
 
-      /*
-      * Bulk methods for adding and removing markers and should be favoured over the
-      * single versions when doing bulk addition/removal of markers.
-      */
-      addLayers(layers: Layer[]): this;
-      removeLayers(layers: Layer[]): this;
-
-      clearLayers(): this;
-
-      /*
-      * If you have a marker in your MarkerClusterGroup and you want to get the visible
-      * parent of it
-      */
-      getVisibleParent(marker: Marker): Marker;
-
-      /*
-      * If you have customized the clusters icon to use some data from the contained markers,
-      * and later that data changes, use this method to force a refresh of the cluster icons.
-      */
-      refreshClusters(clusters?: Marker | Marker[] | LayerGroup): this;
-
-      /*
-      * Returns the total number of markers contained within that cluster.
-      */
-      getChildCount(): number;
-
-      /*
-      * Returns the array of total markers contained within that cluster.
-      */
-      getAllChildMarkers(): Marker[];
-
-      /*
-      * Returns true if the given layer (marker) is in the cluster.
-      */
-      hasLayer(layer: Layer): boolean;
-
-      /*
-      * Zooms to show the given marker (spiderfying if required),
-      * calls the callback when the marker is visible on the map.
-      */
-      zoomToShowLayer(layer: Layer, callback?: () => void): void;
+        /*
+        * Bulk methods for adding and removing markers and should be favoured over the
+        * single versions when doing bulk addition/removal of markers.
+        */
+        addLayers(layers: Layer[]): this;
+        removeLayers(layers: Layer[]): this;
+
+        clearLayers(): this;
+
+        /*
+        * If you have a marker in your MarkerClusterGroup and you want to get the visible
+        * parent of it
+        */
+        getVisibleParent(marker: Marker): Marker;
+
+        /*
+        * If you have customized the clusters icon to use some data from the contained markers,
+        * and later that data changes, use this method to force a refresh of the cluster icons.
+        */
+        refreshClusters(clusters?: Marker | Marker[] | LayerGroup | MarkerCluster | MarkerClusterGroup | {[index: string]: Layer}): this;
```

Comment:
`MarkerCluster` and `MarkerClusterGroup` are unnecessary here as they inherit `Marker` and `LayerGroup`.